### ISSUE TITLE
NaN Loss

### ISSUE THREAD

**Issue by slefcourt27** [cvlab-stonybrook/SelfMedMAE#9]
Hi, 

Thanks for releasing your code publicly.  I have been encountering problems training this using the BTCV dataset as well as my own during training.  I notice that the loss quickly becomes NaN before 100 epochs.  Did you encounter this same issue?  I am using the same library packages, pytorch and python versions.  If you did encounter this during your experiments, will you please share your approach to handling it?

Any help is greatly appreciated!
![Screenshot 2024-04-07 at 12 19 18 AM](https://github.com/cvlab-stonybrook/SelfMedMAE/assets/31667320/75bb2db5-6d96-40b0-a810-f5f0b9ecaed7)

**Comment by MarkonH** [comment]
Hello,

I encountered the same issue while training on my dataset. Do you happen to know the cause or have you found a solution? 

I would greatly appreciate any insights or advice you could share. Thank you very much in advance!
